src/main.py

```diff
@@ -48,20 +48,6 @@
 sys.path.insert(1, comun.USERDAILIESDIR)
 
 
-def select_value_in_combo(combo, value):
```

Comment:
These two functions have been changed to suit the GtkTreeView widget and the new selection mechanism.
They are also moved inside the two class methods where they are referenced. No need to keep them as module functions.